modules/application/src/main/java/edu/northeastern/truthtree/controller/login/Login.java

```diff
@@ -19,7 +22,16 @@ public Login(ILoginService service) {
   }
 
   @RequestMapping(value = "/api/login", method = RequestMethod.POST)
-  public Boolean authenticateUser(@RequestBody String password) {
-    return service.authenticateUser(password);
+  public void authenticateUser(@RequestBody String password, HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
+    Boolean authenticateUser = service.authenticateUser(password);
+    HttpSession httpSession = httpServletRequest.getSession();
+    if(authenticateUser) {
+      httpSession.setAttribute("admin",true);
+      httpServletResponse.setStatus(HttpServletResponse.SC_OK);
+    }
+    else {
+      // wrong login
+      httpServletResponse.setStatus(HttpServletResponse.SC_FORBIDDEN);
```

Comment:
401 code is probably more fitting here instead of 401

Reply:
I guess wrong login is forbidden(403) and after correct login, you access something that you are not authorized to is unauthorized(401)